google/cloud/bigtable/data/_async/_read_rows.py

```diff
@@ -143,70 +162,82 @@ def _read_rows_attempt(self) -> AsyncGenerator[Row, None]:
             retry=None,
         )
         chunked_stream = self.chunk_stream(gapic_stream)
-        return self.merge_rows(chunked_stream)
+        return self.merge_rows(chunked_stream, self._operation_metrics)
 
     async def chunk_stream(
         self, stream: Awaitable[AsyncIterable[ReadRowsResponsePB]]
     ) -> AsyncGenerator[ReadRowsResponsePB.CellChunk, None]:
         """
         process chunks out of raw read_rows stream
         """
-        async for resp in await stream:
-            # extract proto from proto-plus wrapper
-            resp = resp._pb
```

Comment:
diff here is hard to read. I just 1) pulled the call out into its own variable 2) wrapped this in a try/except